_posts/2025-11-12-homebrew-5.0.0.md

```diff
@@ -0,0 +1,84 @@
+---
+title: 5.0.0
+author: MikeMcQuaid
+redirect_from: /blog/5.0.0/
+---
+
+Today, I'd like to announce Homebrew 5.0.0.
+The most significant changes since 4.6.0 are download concurrency by default, official support for Linux ARM64/AARCH64, timescales for deprecating macOS Intel and removing macOS Gatekeeper bypass behaviours.
```

Comment:
```suggestion
The most significant changes since 4.6.0 are download concurrency by default, official support for Linux ARM64/AArch64, timescales for deprecating macOS Intel and removing macOS Gatekeeper bypass behaviours.
```